qmk_firmware/keyboards/keyball/lib/keyball/keyball.c

```diff
@@ -407,6 +407,28 @@ void keyball_oled_render_keyinfo(void) {
 #endif
 }
 
+void keyball_oled_render_layerinfo(void) {
+#ifdef OLED_ENABLE
+    // Format: `Layer: {layer}`
+    //
+    // Output example:
+    //
+    //     Layer: 0
+    //
+    oled_write_P(PSTR("Layer: "), false);
+    layer_state_t layer = layer_state;
+    for (uint8_t i = 0; i < 8; i++) {
+        if (layer & 1) {
+            oled_write_char('#', false);
```

Comment:
```suggestion
            oled_write_char('0' + i, false);
```
完全に個人の感想で申し訳ないのですが、「#」よりレイヤー番号を表示したほうがわかりやすいのではないかと思いました！
「----4---」みたいな感じです！
横からすみません🙇‍♂️

Reply:
確かに分かりやすいです！ありがとうございます